sinabs/backend/dynapcnn/dynapcnn_layer.py

```diff
@@ -1,167 +1,229 @@
+# author    : Willian Soares Girao
```

Comment:
The author and contact tags at the beginning of files are not a practice in our code base. Please remove.